src/xian/methods/query.py

```diff
@@ -157,10 +157,25 @@ async def query(self, req) -> ResponseQuery:
                 except:
                     result = {"stdout": "", "stderr": ""}
 
-            # http://localhost:26657/abci_query?path="/calculate_stamps/<encoded_tx>"
```

Comment:
can we maintain the previous endpoint so this doesn't instantly break any libs pointing to it ?

Reply:
I did make sure that the old endpoint is still there. It's not removed, we have two endpoints now but dependent apps should plan for switching to the new endpoint so that we can after some time remove the old one.